source/features/parse-backticks.tsx

```diff
@@ -25,6 +25,9 @@ function init(): void {
 		'.Box-row.js-navigation-item a.link-gray-dark', // `isGlobalDiscussionList`
 		'[data-channel^="check_suites"] a', // `isActions`
 		'.repository-content .pr-toolbar h2', // `isActions` run
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden a:first-child', // `isRepoRoot` "Repository refresh" layout
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden pre', // `isRepoRoot` "Repository refresh" layout
+		'.js-issue-title', // `isIssue` "Repository refresh" layout
```

Comment:
You can add one more
```suggestion
		'.js-issue-title', // `isIssue` "Repository refresh" layout
		'.existing-pull h4 > a', `isCompare` existing PR "Repository refresh" layout
```

Test on https://github.com/sindresorhus/refined-github/compare/master...yakov116:fix-blanches?expand=1

Reply:
Nice catch! I think `.existing-pull-contents .list-group-item-link` is better.